container-core/src/main/java/ai/vespa/cloud/SystemInfo.java

```diff
@@ -1,5 +1,32 @@
 // Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 package ai.vespa.cloud;
 
+import com.google.inject.Inject;
+import com.yahoo.cloud.config.ConfigserverConfig;
+
+/**
+ * Provides information about the system in which this container is running.
+ * This is available and can be injected when running in a cloud environment.
+ *
+ * @author bratseth
+ */
 public class SystemInfo {
+
+    private final Zone zone;
+
+    /** Do not use */
```

Comment:
I suggest to skip this comment. What does it mean?

Reply:
It means to not use it ... I probably want to switch to another config later which will break any who use this method.